.github/workflows/scorecards-analysis.yml

```diff
@@ -22,7 +22,8 @@ concurrency:
   cancel-in-progress: true
 
 # Declare default permissions as read only.
-permissions: read-all
```

Comment:
I believe this sets the default permission to read-all for all scopes, which is what I think we want.  I think the change below will just set default for contents.  Can we change both this file and codeql.yaml to "permissions: read-all" and re-run the jobs to see if that works? 

https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

Reply:
okay